src/databricks/labs/ucx/assessment/jobs.py

```diff
@@ -116,12 +117,18 @@ def _prepare(all_jobs) -> tuple[dict[int, set[str]], dict[int, JobInfo]]:
             job_name = job_settings.name
             if not job_name:
                 job_name = "Unknown"
+            if job.settings.tasks:
+                spark_jar_job = [task for task in job.settings.tasks if task.spark_jar_task is not None]
+            else:
+                spark_jar_job = None
+            jar_fg = 1 if spark_jar_job else 0
```

Comment:
do this logic the way it's reusable in `JobsCrawler` and `SubmitRunsCrawler`. Probably in `JobsMixin`.